### DISCUSSION TITLE
Export logs to XML

### DISCUSSION THREAD

**Discussion by pinamiranda** [Keyfactor/ejbca-ce#143] [Q&A] [answered]
Hi,
In cesecore.properties I can configure to export logs to XML.
My configuration is the following and I would expect to get a XML log file in /tmp directory, but never happens:
> securityeventsaudit.implementation.1=org.cesecore.audit.impl.integrityprotected.IntegrityProtectedDevice
> securityeventsaudit.exporter.1=org.cesecore.audit.impl.AuditExporterXml
> securityeventsaudit.deviceproperty.1.export.dir=/tmp/

Is there some additional configuration needed?

Many thanks.
Best regards,
José

**Comment by mike-agrenius-kushner** [comment, marked as answer]
Hi, 

We should probably add some info to the configuration file, but the export comes from the Integrity Protected Audit Log, which as noted [here](https://doc.primekey.com/ejbca/ejbca-operations/ejbca-ca-concept-guide/logging/audit-log-overview/integrity-protected-security-audit-log) is an Enterprise feature. 

Cheers,
Mike
